### PR TITLE
Get background file from `main` in `package.json`

### PR DESCRIPTION
**Pull request by Richienb**
~~Why detect the `tsconfig.json` file when you can just detect the file that is going to be compiled in the first place?~~
Get the background file from the `main` property in the `package.json` file.

Currently I am using an empty `tsconfig.json` file to use `background.ts` without a `tsconfig.json` file.

### CODE REVIEW

File: lib/webpack/helpers.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ const existsSync = (f: string): boolean => {
 };
 
 const cwd = process.cwd();
-const ext = existsSync(path.join(cwd, 'tsconfig.json')) ? '.ts' : '.js';
+const ext = existsSync(path.join(cwd, 'background.ts')) ? '.ts' : '.js';
```
Comment (saltyshiomix) [inline lib/webpack/helpers.ts:16]: [IMO] It would be better to check `package.json#main` property :)

**Review by saltyshiomix — DISMISSED** [review]
Sorry for late reply and thank you for your PR!

It would be nice to check `package.json#main` property because we can rename any entry point :)

**Comment by andirsun** [comment]
@Richienb Can you please explain me what problem this PR solves ? I want to test your pr and if solves or improves something then merge it.

**Comment by Richienb** [comment]
I want to use TypeScript without needing to create an empty tsconfig.json file.

**Comment by andirsun** [comment]
I am going to test this PR tomorrow and see how it goes

**Review by andirsun — APPROVED** [review]
LGTM

**Review by andirsun — APPROVED** [review]

**Comment by alexis-piquet** [comment]
Is it relevant to keep this PR open? I dont think so.

**Comment by Richienb** [comment]
I don't think the problem is solved yet

**Comment by alexis-piquet** [comment]
2 years for a PR its a very long one, this is the reason why I've added a comment previously. ;)

If the issue isn't resolved yet, could you fix conflicts? Without that I can't approve it.

**Comment by andirsun** [comment]
@alexis-piquet please fix the remaining conflict to approve.

**Comment by alexis-piquet** [comment]
Looks like something went wrong with the branch, it's not pushed in the repo properly and I cant fetch it. @Richienb Could you fix conflicts or push it properly? 😘

**Comment by Richienb** [comment]
ok

**Review by alexis-piquet — APPROVED** [review]

**Comment by andirsun** [comment]
Lets merge to see how it goes and then we can iterate or revert if needed.

**Pull request merged**